packages/manager/apps/dedicated/client/app/account/kyc-documents/dashboard/controller.js

```diff
@@ -0,0 +1,55 @@
+import { FRAUD_STATUS } from '../constants';
+import { DOCUMENT_TYPE, DOCUMENT_LIST, MAXIMUM_SIZE } from './constants';
+
+export default class KycDocumentsCtrl {
+  /* @ngInject */
+  constructor($translate, coreURLBuilder) {
+    this.$translate = $translate;
+    this.maximum_size = MAXIMUM_SIZE;
+    this.DOCUMENT_TYPE = DOCUMENT_TYPE;
+    this.coreURLBuilder = coreURLBuilder;
+    this.FRAUD_STATUS = FRAUD_STATUS;
+  }
+
+  $onInit() {
+    // init uploaded documents list
+    this.documents = [];
+
+    // personnalcorporation user is like corporation user
+    if (this.user.legalform === 'personalcorporation')
+      this.user.legalform = 'corporation';
+
+    this.supportLink = this.coreURLBuilder.buildURL(
```

Comment:
We could avoid exposing the builder bu building this URL in the constructor.
But should we ?

Reply:
True I will change this